server/src/service-layer/controllers/AdminController.ts

```diff
@@ -141,6 +148,25 @@ export class AdminController extends Controller {
     }
   }
 
+  @SuccessResponse("200", "Booking deleted successfuly")
+  @Security("Jwt", ["admin"])
+  @Post("/bookings/delete")
+  public async removeBooking(
+    @Body() requestBody: DeleteBookingRequest
+  ): Promise<BookingDeleteResponse> {
+    const { bookingId } = requestBody
+    // Validate and check if the booking actually exists
+    const bookingDataService = new BookingDataService()
+    const { user_id } = await bookingDataService.getBookingById(bookingId)
+    if (!user_id) {
+      this.setStatus(404)
+      return { message: "Booking not found with that booking ID." }
```

Comment:
Nit: early return means you don't need else

Reply:
Ah, yep also noticed that and fixed in the later push